src/components/navigation/section-document-or-browser.tsx

```diff
@@ -33,12 +36,16 @@ export const SectionDocumentOrBrowser: React.FC<IProps> = observer(function Sect
   const queryClient = useQueryClient();
   const user = useUserStore();
   const classStore = useClassStore();
+  const [scrollerCollapsed, setScrollerCollapsed] = useState(false);
+  const [showLeftScrollArrow, setShowLeftScrollArrow] = useState(false);
+  const [showRightScrollArrow, setShowRightScrollArrow] = useState(false);
```

Comment:
I feel like showing the arrows shouldn't be state, it should be derived from state. For example, when the component renders you'll want to check to see if you're all the way to the left, and if you are you don't want to display the left arrow. It looks like you're not using the setters anyway, so maybe you're already on track to doing this.